pulumi/grapl/__main__.py

```diff
@@ -211,6 +212,12 @@ def main() -> None:
         ),
     )
 
+    nomad_server_cluster = NomadServer(
```

Comment:
given my comments on `infra.nomad_server_cluster` this would become

```python
nomad_cluster = NomadCluster(
    "nomad-cluster",
    network=network,
    internal_service_ports=(),
)
```